src/main/java/br/com/labbs/monitor/filter/MetricsCollectorFilter.java

```diff
@@ -43,39 +45,51 @@
     private static final String BUCKET_CONFIG_PARAM = "buckets";
     private static final String PATH_DEPTH_PARAM = "path-depth";
     private static final String EXCLUSIONS = "exclusions";
+    private static final String VERSION = "version";
     private static final String DEBUG = "debug";
     private final List<String> exclusions = new ArrayList<String>();
 
     private int pathDepth = 0;
+    private String version;
 
     @Override
-    public void init(FilterConfig filterConfig) throws ServletException {
+    public void init(FilterConfig filterConfig) {
+
+        this.version = getApplicationVersionFromPropertiesFile();
+
         double[] buckets = null;
         if (filterConfig != null) {
-            if (!isEmpty(filterConfig.getInitParameter(DEBUG))) {
-                DebugUtil.setDebug(filterConfig.getInitParameter(DEBUG));
+            String debugParam = filterConfig.getInitParameter(DEBUG);
+            if (isNotEmpty(debugParam)) {
+                DebugUtil.setDebug(debugParam);
+            }
+            String versionParam = filterConfig.getInitParameter(VERSION);
+            if (isNotEmpty(versionParam)) {
+                this.version = versionParam;
             }
```

Comment:
This parameter isn't in documentation, please add in doc or review the code

Reply:
done